src/scripts/order_data.py

```diff
@@ -2,41 +2,30 @@
 # (C) Júlio Barreto
 # Script para ordenar os dados do data.js por nome
 # Criado em 16/02/2019
+# Atualizado em 03/03/2019
 
-import json, sys, demjson
+import json, os, termcolor
 
-def remove_indentation(str_list):
-	"""
-	Remove a indentação do objeto JavaScript de modo que ele
-	possa ser decodificado normalmente
-	"""
-	return ''.join([i.strip() for i in str_list])
-	
+cwd = os.getcwd()
 
-with open('../lib/data.js', 'r') as datajs:
-	"""
-	Abre o arquivo atual de dados e extrai o json de acronimos
-	"""
-	line_string = ''.join(datajs.readlines())
+def update_file(name):
+    with open(name, 'r') as datajs:
+        """
+        Abre o arquivo atual de dados e extrai o json de acronimos
+        """
+        datafile = json.load(datajs)
 
-	data_start = line_string.index('{')
-	data_end = line_string.index('};') + 1
-	before = line_string[:data_start]
-	after = line_string[data_end:]
+    with open(name, 'w') as newdatajs:
+        """
+        Exporta o arquivo de dados de modo que as siglas estejam ordenadas
+        """
+        json_text = json.dumps(datafile, sort_keys=True, indent=4, ensure_ascii=False).encode('utf8')
+        newdatajs.write(json_text)
 
-	data_string = line_string[data_start : data_end].split('\n')
-	formatted_data = remove_indentation(data_string)
-	
-	data = demjson.decode(remove_indentation(data_string))
+path_to_data = '/src/lib/%s.json'
+data_files = ['cursos', 'disciplinas', 'girias', 'locais', 'outros']
 
-with open('../lib/data.js', 'w') as newdatajs:
-	"""
-	Exporta o arquivo de dados de modo que as siglas estejam ordenadas
-	"""
-	ordered_data = json.dumps(data, sort_keys=True, indent=4, ensure_ascii=False).encode('utf8')
+termcolor.cprint('Reordenando arquivos de dados', color='yellow', attrs=['bold'])
```

Comment:
Vi que `termcolor` não é padrão e tem que ser instalado, talvez uma melhor opção seja usar [ANSI escape](http://ozzmaker.com/add-colour-to-text-in-python/) code do python para imprimir colorido e com estilo

```suggestion
print('\033[1;93m Reordenando arquivos de dados \033[0m')
```

Reply:
Massa, como não precisei instalar antes, achei que era default, mas sendo assim, melhor mudar pro ansi mesmo